api/v1beta1/kafkacluster_types.go

```diff
@@ -373,6 +385,29 @@ func (c IngressServiceSettings) GetServiceType() corev1.ServiceType {
 	return c.ServiceType
 }
 
+// Replace {{.Id}} in brokerHostnameTemplate with actual broker id
```

Comment:
Suggestion to do string replace here instead of template engine from Go.
Reasons are that we are doing the same process here (not calling any extra functions to require template engine), current deployment spec is done via Helm (need to escape double templating).
wdyt?